test/clj/rems/test_browser.clj

```diff
@@ -562,7 +571,11 @@
   (testing "submit test data with API"
     (btu/context-assoc! :form-id (test-helpers/create-form! {:form/fields [{:field/title {:en "description" :fi "kuvaus" :sv "rubrik"}
                                                                             :field/optional false
-                                                                            :field/type :description}]}))
+                                                                            :field/type :description}
+                                                                           {:field/title {:en "private" :fi "fi" :sv "sv"}
```

Comment:
These days I tend to use the pattern "private en", "private fi", ... or thereabouts for test texts.

Reply:
changed